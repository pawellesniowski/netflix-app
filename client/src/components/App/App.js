import React from 'react';
import ErrorBoundary from "../ErrorBoundary/ErrorBoundary";
import PageLayout from '../PageLayout/PageLayout';

import './App.scss';
import DetailsPage from "../../pages/details-page/DetailsPage";
import HomePage from "../../pages/home-page/HomePage";
import Header from "../Header/Header";

const App = () => {
  return (
    <ErrorBoundary>
      <PageLayout>
        <Header />
        {/*<HomePage/>*/}
        <DetailsPage/>
      </PageLayout>
    </ErrorBoundary>
  )
};

export default App;
