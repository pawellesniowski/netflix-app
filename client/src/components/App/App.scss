@font-face {
  font-family: "Bebas Neue";
  font-style: normal;
  font-weight: 400;
  src: url("../../common/styles/fonts/bebas-neue.ttf");
}

* {
  font-family: "Bebas Neue", serif;;
}
