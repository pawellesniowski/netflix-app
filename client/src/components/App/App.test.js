import React from 'react';
import App from './App';
import {shallow} from "enzyme"

describe('App Component', () => {
    let component;
    beforeEach(() => {
        component = shallow(<App/>);
    });

    it('Renders App Component Snapshot', () => {
        expect(component).toMatchSnapshot();
    });
});
