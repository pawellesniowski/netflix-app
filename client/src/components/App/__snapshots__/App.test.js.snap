// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Component Renders App Component Snapshot 1`] = `ShallowWrapper {}`;
